lib/base/scriptutils.cpp

```diff
@@ -513,10 +513,13 @@ String ScriptUtils::MsiGetComponentPathShim(const String& component)
 	path[0] = '\0';
 	MsiGetComponentPath(productCode, component.CStr(), path, &szPath);
 	return path;
+}
 #else /* _WIN32 */
+String ScriptUtils::MsiGetComponentPathShim(const String&)
+{
```

Comment:
Why should this function exist on non-Windows in the first place?

Reply:
Would you be fine with not having `System.msi_get_component_path()` on \*nix?

Reply:
Wait, that's exposed there? Okay, then keep it.